components/automate-ui/src/app/page-components/feature-flags/feature-flags.component.ts

```diff
@@ -39,6 +39,7 @@ export class FeatureFlagsComponent implements OnInit {
 
   public warning: string;
   public flagType: FlagTypes;
+  private blockedElements = ['input', 'textarea', 'button', 'select', 'chef-button', 'chef-select'];
```

Comment:
No - there's no real example I can think of specifically where this does interfere.  My attempt was to play it safe and think of any inputs that could possibly have some sort of interaction, but I can't say there was a specific area of our site that it would cause a problem.  I'll remove those ones 👍